### ISSUE TITLE
Crashed: com.Instabug.queueUsageLoggerQueue

### ISSUE THREAD

**Issue by TinyQ** [Instabug/Instabug-iOS#245]
### Steps to Reproduce the Problem
![image](https://user-images.githubusercontent.com/3217732/39718739-c83452c4-5269-11e8-927d-274b77ef7d31.png)

[com.zhihu.ios_issue_75324_crash_d21d421bed2d4a32a2d8ebd467df606e_DNE_0_v2.txt](https://github.com/Instabug/Instabug-iOS/files/1981057/com.zhihu.ios_issue_75324_crash_d21d421bed2d4a32a2d8ebd467df606e_DNE_0_v2.txt)

### SDK Version
7.12.5

**Comment by Kmohamed** [comment]
Are you this crash from sdk version 7.12.5 ?

**Comment by Kmohamed** [comment]
@TinyQ @LFabien 
We are very sorry for experience this issue again.
This issue is caused from calculating cpu usage of dispatch queues.
What we are doing here is getting all threads from `mach_task_self()` and calculating the cpu usage for each thread. As you know that each thread is assigned to a single dispatch_queue and there is a property in thread called `dispatch_qaddr` which is the address of `dispatch_queue` for this thread.
The crash occurs when we are trying to dereference this address and get the `dispatch_queue`.
we put a lot of safety code as much as we can to prevent this crash like check if this is a valid pointer but we failed. 
So the thing that’s most important for you is the stability of the SDK and hence we decided to disable it for now in SDK version (7.12.6) for a while and open DTS to Apple and I will keep you updated.
